README.md

```diff
@@ -16,14 +16,10 @@ pip install -e .
 
 ### Running the command line tool
 
-Call `libcovebods` and pass the filename of some JSON data.
+Call `libcovebods`.
 
-    libcovebods tests/fixtures/0.1/basic_1.json
+    libcovebods -h tests/fixtures/0.1/basic_1.json
```

Comment:
I find `-h` with a filename kind of odd here.

Reply:
Remove tests/fixtures/0.1/basic_1.json and leave -h